### PR TITLE
Minor tweaks to self-managed on-prem

### PR DESCRIPTION
**Pull request by caseydavenport**
A few typo corrections and other minor wording tweaks.

- [ ] Tests
- [ ] Documentation
- [ ] Release note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 801a736b2230732b37b65b24a5e9dc764249279d

https://deploy-preview-3160--calico.netlify.com

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 1]
```diff
@@ -22,28 +22,28 @@ This how-to guide uses the following {{site.prodname}} features:
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. We recommend [Customizing Calico manifests ({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes; this avoids downstream manual updates to other {{site.prodname}} resources.
+{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
```
Comment (bmckercher123) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]: you cannot use {{site.prodname}} in a link....unless you know about an update that Turk made?

Comment (caseydavenport) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: Strange, I thought I had done this before.

Comment (caseydavenport) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: It seems to work in the deploy preview: https://deploy-preview-3160--calico.netlify.com/master/getting-started/kubernetes/self-managed-onprem/onpremises

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 2]
```diff
@@ -22,28 +22,28 @@ This how-to guide uses the following {{site.prodname}} features:
...
+### How to
 
 - [Determine your datastore](#determine-your-datastore)
 - [Install Calico on nodes](#install-calico-on-nodes)
 
-#### Determine your datastore 
+#### Determine your datastore
 
-{{site.prodname}} supports both **Kubernetes API datastore (kdd)** and **etcd** datastores. The **Kubernetes API datastore** is preferred for on-premises deployments, and supports only Kubernetes workloads; **etcd** is the best datastore for hybrid deployments. 
+{{site.prodname}} supports both **Kubernetes API datastore (kdd)** and **etcd** datastores. The **Kubernetes API datastore** is the preferred for on-premises deployments, and supports only Kubernetes workloads; **etcd** is the best datastore for hybrid deployments.
```
Comment (lmm) [inline getting-started/kubernetes/self-managed-onprem/onpremises.md]: I think either:

The **Kubernetes API datastore** is preferred for on-premises deployments

or

The **Kubernetes API datastore** is the preferred datastore for on-premises deployments

are better.

File: getting-started/kubernetes/self-managed-onprem/onpremises.md (hunk 1 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: Yipee. Glad it works now!

**Pull request merged**